internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -910,6 +912,7 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				Type:     pluginsdk.TypeBool,
 				Optional: true,
 				ForceNew: true,
+				Default:  false,
```

Comment:
@tombuildsstuff `ApiServerAccessProfile` is not returned when having 'default' settings anymore, which means I'm setting values by default now below. Does this mean I should add `Default: true` to the schema? Does this change requires a state migration?

Or is there a simpler solution?

Reply:
Taking a look through here this should be fine (but unnecessary) since this implicitly defaults to `false` in the read function?

Reply:
Indeed, it implicitly defaults to 'false' in the read function, which is a change. I made it here explicit.

The main question is whether this defaulting (implicit or explicit) is a breaking change which requires a state migration, as this is also a `ForceNew` setting.